models/chainlink/arbitrum/chainlink_arbitrum_ccip_fulfilled_transactions.sql

```diff
@@ -14,26 +13,46 @@
 WITH
   ccip_fulfilled_transactions AS (
     SELECT
-      ccip_send_traces.tx_hash as tx_hash,
-      ccip_send_traces.block_time as block_time,
-      cast(date_trunc('day', ccip_send_traces.block_time) as date) as date_start,
-      ccip_send_traces."from" as "node_address",
-      ccip_send_traces.trace_address as trace_address
+      ccip_send_logs_v1.tx_hash as tx_hash,
+      ccip_send_logs_v1.block_time as block_time,
+      cast(date_trunc('day', ccip_send_logs_v1.block_time) as date) as date_start,
+      ccip_send_logs_v1.tx_from as caller_address,
+      ccip_send_logs_v1.tx_index as tx_index
     FROM
-      {{ ref('chainlink_arbitrum_ccip_send_traces') }} ccip_send_traces
+      {{ ref('chainlink_arbitrum_ccip_send_requested_logs_v1') }} ccip_send_logs_v1
+      LEFT JOIN {{ source('arbitrum', 'transactions') }} tx ON
+        ccip_send_logs_v1.tx_hash = tx.hash
```

Comment:
```suggestion
        ccip_send_logs_v1.tx_hash = tx.hash
        {% if is_incremental() %}
            AND tx.block_time >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
        {% endif %}
```

Reply:
i would continue to filter incrementally in join conditions to match filter on main table in from statement, especially on these large raw base tables

Reply:
looks like two spots to apply in this spell, then two spots in reverted as well, then of course all other chains too

Reply:
I added the increments in both spells